src/surfaces/actionBar/actionBar.jsx

```diff
@@ -84,7 +89,7 @@ const defaultProps = {
 };
 
 const HAS_DRAWER_ACTION_BAR_CLASS_NAME = 'drawer-has_action_bar';
-const HAS_PAGE_ACTION_BAR_CLASS_NAME = 'page-has_action_bar';
+const HAS_PAGE_ACTION_BAR_CLASS_NAME = 'page-has_action\_bar';
```

Comment:
Is that back slash intended there?

Reply:
Nope, that is not intended. Thanks for the catch.

Reply:
Lookin' better.  Resolving.